doc/costs.rst

```diff
@@ -8,7 +8,7 @@ Cost Assumptions
 
 The database of cost assumptions is retrieved from the repository
 `PyPSA/technology-data <https://github.com/pypsa/technology-data>`_ and then
-saved to ``resources/costs.csv``. Cost assumptions of previous PyPSA-Eur
+saved to ``resources/costs.csv``. Cost assumptions of previous PyPSA-Earth
 versions can be restored by using setting retrieve_costs in the config to false.
```

Comment:
Might be worth to revise the statement here. I am not sure it is clear what cost assumption of previous PyPSA-Earth versions is.